redis_enterprise/assets/dashboards/redis-enterprise-database-dashboard.json

```diff
@@ -0,0 +1,851 @@
+{
+    "title": "Redis Enterprise Database Dashboard (JS)",
+    "description": "",
```

Comment:
Dashboard is missing a description, see [best practices guidelines](https://docs.datadoghq.com/developers/integrations/create-an-integration-dashboard?tab=ui#follow-dashboard-best-practices).